application/src/main/resources/application.yml

```diff
@@ -42,10 +42,19 @@ springdoc:
 #server:
 #  ssl:
 #    enabled: false
+#    client-auth: <client authentication>
+# SSL configuration using certificate stores
 #    key-store: <path to keystore>
 #    key-store-password: <password>
 #    key-store-type: <keystore type>
 #    key-alias: <key alias>
 #    key-password: <key password>
-# Rate at which certificate are reloaded automatically
+# SSL configuration using certificates in PEM format
+#    certificate: <path to certificate>
+#    certificate-private-key: <path to certificate private key>
+#    key-store-password: <key password>
+# Optional configuration for certificates in PEM format
+#    trust-certificate: <path to certificate>
+#    trust-certificate-private-key: <path to certificate key>
```

Comment:
trust-certificate-private-key - when would you ever use this? It feels strange to have private key for the CAs since the owner of the cerificate usually doesn't own the private key of the CAs.

Reply:
It's a possible configuration according to the tool docs. It can be removed.